core/src/test/scala/com/nvidia/spark/rapids/tool/profiling/AutoTunerSuite.scala

```diff
@@ -210,6 +210,7 @@ class AutoTunerSuite extends FunSuite with BeforeAndAfterEach with Logging {
           |--conf spark.rapids.sql.concurrentGpuTasks=2
           |--conf spark.rapids.sql.multiThreadedRead.numThreads=20
           |--conf spark.shuffle.manager=com.nvidia.spark.rapids.spark311.RapidsShuffleManager
+          |--conf spark.sql.adaptive.coalescePartitions.minPartitionNum=1
```

Comment:
wouldn't this fail if ran on spark versions >= 3.2?

Reply:
No. The version of Spark comes for the application event log, not from the version of spark that is on the classpath of the tool. These tests explicitly mock out the version of Spark to 3.1.1 to match dataproc 2.0

Reply:
so do we want to make sure we print the Spark version that these configs go with just in case they switch versions?  Or at least document user should run explicitly the same version so should rerun when switching versions.

Reply:
ok, lets just leave it and deal with it later if we start recommending the newer configs